src/nbl/asset/ICPUDescriptorSet.cpp

```diff
@@ -125,11 +125,11 @@ IAsset* ICPUDescriptorSet::getDependant_impl(size_t ix)
 				case IDescriptor::EC_SAMPLER:
 					return static_cast<ICPUSampler*>(desc);
 				case IDescriptor::EC_IMAGE:
-					return static_cast<ICPUImage*>(desc);
+					return static_cast<ICPUImageView*>(desc);
```

Comment:
@Fletterio @deprilula28 @Erfan-Ahmadi I have no clue how our code worked for like 2 years with this 🤣